docs/maui/layouts/DockLayout.md

```diff
@@ -0,0 +1,121 @@
+---
+title: DockLayout - .NET MAUI Community Toolkit
+author: profix898
+description: The DockLayout is a layout where views can be docked to the sides (top, left, right, bottom) of the layout container.
+ms.date: 11/04/2022
+---
+
+# DockLayout
+
+`DockLayout` is a layout where views can be docked to the sides of the layout container.
+
+The image below shows how a `DockLayout` is conceptually structured. Child views are docked at one of 4 possible docking positions: *Top*, *Bottom*, *Left* or *Right* (equivalent to `DockPosition.Top`, `DockPosition.Bottom`, `DockPosition.Left`, and `DockPosition.Right`). Views that are not explicitly docked (or with `DockPosition.None`) are displayed at the center (or between *Top* / *Bottom* and *Left* / *Right* positions).
+
+![DockLayout concept](/docs/images/layouts/DockLayout.svg)
```

Comment:
Let's try a relative path

```suggestion
![DockLayout concept](../images/layouts/DockLayout.svg)
```

Reply:
Ignore this I see you are already on the case

Reply:
The image was in a different 'images' folder. Moved it and updated to use relative path.